guidelines/index.html

```diff
@@ -1746,24 +1746,24 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
+  <p><a href="#dfn-web-page-s" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-web-page-s-9" title="単一の URI から HTTP で得た埋め込まれていないリソースに加え、レンダリングに使われる、又はユーザエージェントがこのリソースと一緒にレンダリングすることを意図しているその他のあらゆるリソースを合わせたもの。">ウェブページ</a>が以下のいずれかのヘルプの<a href="#dfn-mechanism" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-mechanism-12" title="結果を得るためのプロセス又は手法。">メカニズム</a>を含む場合、かつ、それらのメカニズムが<a href="#dfn-set-of-web-pages" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-set-of-web-pages-5" title="共通の目的を共有し、同じコンテンツ制作者、グループ、又は組織により制作されたウェブページの集合。">ウェブページ一式</a>の中の複数のウェブページで繰り返される場合、利用者による変更がなされない限り、それらは他のページコンテンツに対して相対的に同じ順序で出現する。</p>
 
     <ul>
-        <li>Human contact details;</li>
-        <li>Human contact mechanism;</li>
-        <li>Self-help option;</li>
-        <li>A fully automated contact mechanism.</li>
+        <li>人間がコンタクトする詳細情報</li>
+        <li>人間がコンタクトするメカニズム</li>
+        <li>セルフヘルプのオプション</li>
+        <li>完全に自動化されたコンタクトのメカニズム</li>
     </ul>
 
-  <div class="note" role="note" id="issue-container-generatedID-39"><div role="heading" class="note-title marker" id="h-note-39" aria-level="5"><span>注記 1</span></div><p class="">Help mechanisms may be provided directly on the page, or may be provided via a direct link to a different page containing the information.</p></div>
-  <div class="note" role="note" id="issue-container-generatedID-40"><div role="heading" class="note-title marker" id="h-note-40" aria-level="5"><span>注記 2</span></div><p class="">For this Success Criterion, "the same order relative to other page content" can be thought of as how the content is ordered when the page is serialized. The visual position of a help mechanism is likely to be consistent across pages for the same page variation (e.g., CSS break-point). The user can initiate a change, such as changing the page's zoom or orientation, which may trigger a different page variation. This criterion is concerned with relative order across pages displayed in the same page variation (e.g., same zoom level and orientation).</p></div>
+  <div class="note" role="note" id="issue-container-generatedID-39"><div role="heading" class="note-title marker" id="h-note-39" aria-level="5"><span>注記 1</span></div><p class="">ヘルプのメカニズムは、ページ上で直接提供されることもあれば、情報を含む別のページへの直接リンクを通じて提供されることもある。</p></div>
+  <div class="note" role="note" id="issue-container-generatedID-40"><div role="heading" class="note-title marker" id="h-note-40" aria-level="5"><span>注記 2</span></div><p class="">この達成基準における「他のページコンテンツに対して相対的に同じ順序」は、ページが線形化されたときにコンテンツがどう順序づけられるか、として考えることができる。視覚的なヘルプのメカニズムの位置は、同じページバリエーション (例えば CSS のブレークポイント) においてページ間で一貫している可能性が高い。利用者は、ページのズームあるいはデバイスの向を変えるなどして変更を発生させることができ、それによって異なるページバリエーションが引き起こされる可能性がある。この達成基準は、同じページバリエーション (例えば、同じズームレベルであったり、同じ向きであったり) で表示されるページ間の、相対的な順序を関心事としている。</p></div>
```

Comment:
thought of as ～（である）と考えられる
<https://eow.alc.co.jp/search?q=thought+of+as>

variation　差異
<https://eow.alc.co.jp/search?q=variation>

concerned with ～に関心がある　としてみました
<https://eow.alc.co.jp/search?q=concerned+with>

```suggestion
  <div class="note" role="note" id="issue-container-generatedID-40"><div role="heading" class="note-title marker" id="h-note-40" aria-level="5"><span>注記 2</span></div><p class="">この達成基準における「他のページコンテンツに対して相対的に同じ順序」は、ページが線形化されたときにコンテンツが順序付けられる方法と考えることができる。ヘルプのメカニズムの視覚的な位置は、同じページの差異 (例えば CSS のブレークポイント) に対してページ間で一貫している可能性が高い。利用者は、ページのズーム、ページの向きを変えるなどの変更を発生させることができ、これによって異なるページの差異を引き起こす可能性がある。この達成基準は、同じページの差異 (例えば、同じズームレベル、向き) で表示されるページ間の相対的な順序に関心がある。</p></div>
```